less/bootstrap-slider.less

```diff
@@ -151,3 +162,31 @@
 		}
 	}
 }
+.slider-tick {
+	position: absolute;
+	width: @slider-line-height;
+	height: @slider-line-height;
+	#gradient > .vertical(#f9f9f9, #f5f5f5);
```

Comment:
Is this #gradient selector necessary? Also want to refrain from using IDs in stylesheets.